### DISCUSSION TITLE
CoCoTb_Bus.BUS is deprecated in latest release. Is their an equivalent method for it?

### DISCUSSION THREAD

**Discussion by Eulerianial** [cocotb/cocotb#3304] [Q&A]

**Comment by Eulerianial** [comment]
I was trying the SPI library for the cocotb i.e. cocotbext-spi.
There, an import for cocotb_bus.bus was referrred.
But it seems this one is deprecated. 
Is there an equivalent of this in the latest release.

**Comment by ktbarrett** [comment]
It doesn't appear to be deprecated? https://github.com/cocotb/cocotb-bus/blob/master/src/cocotb_bus/bus.py

Try updating your versions of cocotb and the SPI library. Or give more context.
